pkgs/applications/audio/tenacity/default.nix

```
@@ -0,0 +1,156 @@
+{ stdenv
+, lib
+, fetchgit
+, cmake
+, wxGTK
+, pkg-config
+, python3
+, gettext
+, glib
+, file
+, lame
+, libvorbis
+, libmad
+, libjack2
+, lv2
+, lilv
+, makeWrapper
+, serd
+, sord
+, sqlite
+, sratom
+, suil
+, alsa-lib
+, libsndfile
+, soxr
+, flac
+, twolame
+, expat
+, libid3tag
+, libopus
+, ffmpeg
+, soundtouch
+, pcre
+, portaudio
+, linuxHeaders
+, at-spi2-core
+, dbus
+, epoxy
+, libXdmcp
+, libXtst
+, libpthreadstubs
+, libselinux
+, libsepol
+, libxkbcommon
+, util-linux
+}:
+
+let
+  inherit (lib) optionals;
+
+in
+stdenv.mkDerivation rec {
```

```suggestion

stdenv.mkDerivation rec {
```
Please use lib.optionals.

---

Done.